src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -673,7 +708,7 @@
 
 helps['role'] = """
 type: group
-short-summary: Manage user roles for access control with Azure Active Directory and service principals.
+short-summary: Azure role-based access control (Azure RBAC).
```

Comment:
The previous statement doesn't even make sense.

Also, I feel `Manage Azure role-based access control` too verbose as "manage" and "control" are very similar concepts.

Reply:
The short-summary needs to start with a verb.  Can we put "manage" back in the description here?

Reply:
@jiasli, Why are we taking out "what kind of user" (AAD)? Shouldn't we at least say "Azure user"?

Reply:
> The short-summary needs to start with a verb. Can we put "manage" back in the description here?

Should we use

```suggestion
short-summary: Manage Azure role-based access control (Azure RBAC).
```

Reply:
> Why are we taking out "what kind of user" (AAD)? Shouldn't we at least say "Azure user"?

I explained in the PR description https://github.com/Azure/azure-cli/pull/27756#issue-1976129746 and https://github.com/Azure/azure-cli/pull/27756#discussion_r1384326021.